subprojects/chrome-trace/src/main/java/org/gradle/trace/monitoring/GCMonitoring.java

```diff
@@ -38,7 +38,6 @@ public void start(TraceResult traceResult) {
                 if (notification.getType().equals(GarbageCollectionNotificationInfo.GARBAGE_COLLECTION_NOTIFICATION)) {
                     GarbageCollectionNotificationInfo info = GarbageCollectionNotificationInfo.from((CompositeData) notification.getUserData());
                     //get all the info and pretty print it
-                    long duration = info.getGcInfo().getDuration();
```

Comment:
Unused variable: `duration`. Let me know if it should have been part of some trace result.